productivity-suite/piercing-library/src/piercing-fragment-outlet.ts

```diff
@@ -76,16 +79,14 @@ export class PiercingFragmentOutlet extends HTMLElement {
   }
 
   disconnectedCallback() {
-    unmountedFragmentIds.add(this.fragmentId);
-  }
-
-  private pierceFragmentIntoDOM(fragmentHost: PiercingFragmentHost) {
-    this.innerHTML == "";
-    fragmentHost.pierceInto(this);
+    if (this.fragmentHost) {
```

Comment:
If there is no `fragmentHost` then we never completed the loading of the fragment.